crates/solver/src/solver/oneinch_solver.rs

```diff
@@ -114,6 +114,9 @@ impl OneInchSolver {
         tracing::debug!("querying 1Inch swap api with {:?}", query);
         let swap = match self.client.get_swap(query).await? {
             RestResponse::Ok(swap) => swap,
+            RestResponse::Err(error) if error.description == *"insufficient liquidity" => {
```

Comment:
I'm surprised that the `*` before the string comparison is needed. Did you try without it?